src/main/msp/msp.c

```diff
@@ -1776,6 +1776,14 @@ static bool mspProcessOutCommand(int16_t cmdMSP, sbuf_t *dst)
 #else
         sbufWriteU8(dst, 0);
 #endif
+#if defined(USE_GYRO_DATA_ANALYSE)
+        // Added in MSP API 1.44
```

Comment:
No need for the extra comment as there's already a comment for stuff added in 1.44 above.